tests/test_cli.py

```diff
@@ -414,6 +414,19 @@ def test_echo_unknown_extension_error(tmpdir, cli_runner):
     assert 'Unable to load extension: ' in result.output
 
 
+def test_local_extension(tmpdir, cli_runner):
```

Comment:
Would it be good to include tests for non-happy path? I can think of two cases, but maybe there are more:
- I include a filter in my template, but didn't include the extension in the `_local_extensions` list
- I include a filter in my template and extension in the `_local_extensions` list, but haven't defined that extension (or it's somehow not available to be imported)
  - It looks like there's a [similar test](https://github.com/cookiecutter/cookiecutter/blob/master/tests/test_environment.py#L8:L15) for built-in Jinja extensions

Reply:
that's a good idea. i'll pick that up ASAP